.github/workflows/fix-prettier.yaml

```
@@ -0,0 +1,88 @@
+        run: git push origin ${{ fromJson(steps.get-pr.outputs.result).head_ref }}
+
+      - name: Comment on PR
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const { owner, repo, number } = context.issue;
+            const changes_made = '${{ steps.commit.outputs.changes_made }}' === 'true';
+            const message = changes_made
+              ? '✅ Prettier issues fixed and committed to this PR!'
+              : '✅ No prettier issues found in this PR!';
+            
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)